modules/viewGlobals.js

```diff
@@ -15,6 +25,83 @@ function getMetaTitle(base, pageTitle) {
     }
 }
 
+function getHtmlClasses({ locale, highContrast }) {
+    let parts = ['no-js', 'locale--' + locale];
+
+    if (highContrast) {
+        parts.push('contrast--high');
+    }
+
+    return parts.join(' ');
+}
+
+/**
+ * buildUrl
+ * URL helper, return canonical URL based on sectionName or pageName
+ * Handle fallbacks for toplevel pages or linking to direct paths.
+ */
+function buildUrl(localePrefix) {
```

Comment:
I'm not a fan of this function, feels hacky and brittle, but I think the more we move to the CMS, the less we'll need it.

Reply:
Agreed, it took me a while to even test this function as I'm not 100% sure what the second part of the is _really_ intended for. There's room for the first part, where we link to a named route. It is better to test the brittle thing lest it hang around — Aristotle or something.